packages/clay-management-toolbar/src/ClayResultsBar.soy

```diff
@@ -80,35 +79,58 @@
 {/template}
 
 /**
- * This renders the list of filter labels
- */
-{template .filterLabels}
-	{@param filterLabels: list<[
-		closeable: bool,
-		label: html|string
-	]>}
+ *
+*/
+{template .filterLabel}
+	{@param index: number}
+	{@param label: html|string}
 	{@param spritemap: string}
 	{@param? _handleFilterLabelCloseClicked: any}
+	{@param? closeable: bool}
+	{@param? expand: bool}
+
+	{let $elementClasses kind="text"}
+		tbar-item
+		{if $expand}
+			{sp}tbar-item-expand
+		{/if}
+	{/let}
 
-	<li class="tbar-item tbar-item-expand">
+	<li class="{$elementClasses}">
 		<div class="tbar-section">
-			{foreach $filterLabel in $filterLabels}
-				{call ClayLabel.render}
-					{param closeable: $filterLabel.closeable ?: true /}
-					{param events: ['close': $_handleFilterLabelCloseClicked] /}
-					{param label kind="text"}
-						{$filterLabel.label}
-					{/param}
-
-					{param ref: 'label' + index($filterLabel) /}
-					{param size: 'lg' /}
-					{param spritemap: $spritemap /}
-				{/call}
-			{/foreach}
+			{call ClayLabel.render}
+				{param closeable: $closeable /}
+				{param elementClasses: 'component-label tbar-label' /}
+				{param events: ['close': $_handleFilterLabelCloseClicked] /}
+				{param label: $label /}
```

Comment:
I think we have to wrap this in a `.tbar-section`.

```soy
{param label kind="text"}
    <div class="label-section">{$label}</div>
{/param}
```

Reply:
If you mean in a `label-section` I didn't add it 'cause that class does nothing and changes how the label component is supposed to work.

Reply:
An empty block level element should be there because of `label-item` uses `flex-direction: column` and causes anything that is a direct descendant to become a block level element. As soon as you try and set elements inline, something like `Category: <strong>Productivity</strong>`, you will see it stacks by default.

The class does nothing, but gives something for devs to hook on to and I think it's better than an empty div.

Reply:
hey @pat270. Is there any problem for us to go without it?

Reply:
@matuzalemsteles No problem without, but I see stuff like `.some-class > div` in portal and don't like it.

Reply:
Sorry @pat270 I had not seen your previous comment when I posted it 😅. (I think it did not update for me, so I deleted it...).

So we'd better get involved with the `.label-section`, so we do not give the possibility that others might do `.some-class> div` to try to fix this.

Reply:
Ok guys, I'll add it tomorrow, thx!